mautic/mautic
# Bug zoho mapping

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | #3651
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

Fields with spaces did not correctly populate to Zoho. This PR fixes that. It also removes unsupported features from the list until they are implemented (#3651)

#3674 introduced a bug that wasn't caught that wiped out fields for other integrations. This also addresses that. 

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Map Zoho last name field
2. Create a campaign or form submit action to push to the integration
3. Execute the campaign 
4. View zoho and note that "unknown" will be populated for last name

#### Steps to test this PR:
1. Repeat the above 
2. last name should be correct
3. View the features tab of the config and trigger push should be the only feature listed
